src/adapters/gam/managers/orders.py

```diff
@@ -393,19 +481,24 @@ def log(msg):
                 # Validate format types against product supported types
                 supported_format_types = impl_config.get("supported_format_types", ["display", "video", "native"])
 
-                for format_id in package.format_ids:
+                for format_id_obj in package.format_ids:
```

Comment:
This should have a string printer because just saying format_id_str below won't work if someone uses a different creative agent than the default, because we won't know if it's the default one or a different one, which you could do for generative rendering or whatever. 

We should generally say, "creative.adcxtp.org: display 728 by 90", not just "display 728 by 90."